catkin_ws/src/planner/src/substates/utility/controller.py

```diff
@@ -4,7 +4,7 @@
 import actionlib
 from geometry_msgs.msg import Pose, Vector3, Vector3Stamped
 from std_msgs.msg import Float64, Bool, Header
-from auv_msgs.msg import StateAction, StateGoal, SuperimposerAction, SuperimposerGoal
+from auv_msgs.msg import StateAction, StateGoal, SuperimposerAction, SuperimposerGoal, QuaternionAction, QuaternionGoal
```

Comment:
do we still need all these imports if we're just using the quaternion server?

Reply:
We haven't fully transitioned from euler angles yet. Testing on clarke is necessary for us to deprecate the old code

Reply:
we shouldn't have this intermediate state though, it's gonna be so messy and impossible to debug/test with

Reply:
we should have this on a seperate branch and have no trace of euler angles on that branch, or save the euler angles version of noetic to another branch and remove any trace of them on noetic

Reply:
I don't see how thats the case. We only added one function to the planner that you can just ignore